modules/Microsoft.App/managedEnvironments/deploy.bicep

```diff
@@ -0,0 +1,197 @@
+@description('Required. Name of the Container Apps Managed Environment.')
+param name string
+
+@description('Optional. Existing Log Analytics Workspace resource ID.')
+param logAnalyticsWorkspaceResourceId string = ''
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Tags of the resource.')
+param tags object = {}
+
+@description('Optional. Array of role assignment objects that contain the \'roleDefinitionIdOrName\' and \'principalId\' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: \'/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11\'.')
+param roleAssignments array = []
+
+@allowed([
+  'Consumption'
+  'Premium'
+])
+@description('Optional. Managed environment SKU.')
+param skuName string = 'Consumption'
+
+@description('Optional. Logs destination.')
+param logsDestination string = 'log-analytics'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool
+
+@description('Optional. Application Insights connection string used by Dapr to export Service to Service communication telemetry.')
+@secure()
+param daprAIConnectionString string = ''
+
+@description('Optional. Azure Monitor instrumentation key used by Dapr to export Service to Service communication telemetry.')
+@secure()
+param daprAIInstrumentationKey string = ''
+
+@description('Optional. CIDR notation IP range assigned to the Docker bridge, network. Must not overlap with any other provided IP ranges.')
```

Comment:
```suggestion
@description('Optional. CIDR notation IP range assigned to the Docker bridge, network. It must not overlap with any other provided IP ranges and can only be used when the environment is deployed into a virtual network. If not provided, it will be set with a default value by the platform.')
```

Reply:
@AlexanderSehr will like to suggest bringing this parameter closer to the platformReservedCidr and platformReservedDnsIP as they go hand in hand